src/main/java/com/google/sps/data/Contact.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public class Contact {
+  // Represents contact email.
+  private String email;
+  // Represents contact phone number.
```

Comment:
Instead of/in addition to calling out that this is a phone number, tell the reader what to expect here:

A 10 digit US number: e.g. 111 111 1111 or the text equivalent, 800 HELP ALL1

?